airbyte-workers/src/main/java/io/airbyte/workers/general/CheckConnectionWorker.java

```diff
@@ -4,8 +4,8 @@
 
 package io.airbyte.workers.general;
 
+import io.airbyte.config.ConnectorJobOutput;
 import io.airbyte.config.StandardCheckConnectionInput;
-import io.airbyte.config.StandardCheckConnectionOutput;
 import io.airbyte.workers.Worker;
 
-public interface CheckConnectionWorker extends Worker<StandardCheckConnectionInput, StandardCheckConnectionOutput> {}
+public interface CheckConnectionWorker extends Worker<StandardCheckConnectionInput, ConnectorJobOutput> {}
```

Comment:
One thing I don't love about using a common `ConnectorJobOutput` is we lose _some_ type-safety in what these workers are returning, but seems ok as to stick with our existing convention of using POJOs/json schema for defining these outputs.